test-suite/tests/jobconfig

```diff
@@ -132,6 +132,13 @@ program = WANNIER90_WOUT_OK
 inputs_args = ('LaVO3.win', '')
 output = LaVO3.wout
 
+# Lead, 4 lowest states; Fermi surface, print bvec
+[testw90_bvec]
+program = WANNIER90_WOUT_OK
+inputs_args = ('lead.win', '')
+output = lead.wout
```

Comment:
I think this test shouldn't check the .wout, but rather we should add a new `program` and a new `parser` to check the check the new `bvec` file (see e.g. what I did in #177 for a different file). You can then check with the parser, e.g., the number of bvecs or even all values.